speech/snippets/auto_punctuation.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package snippets
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	// [START imports]
+	"golang.org/x/net/context"
+
+	speech "cloud.google.com/go/speech/apiv1"
+	speechpb "google.golang.org/genproto/googleapis/cloud/speech/v1"
+	// [END imports]
+)
+
+func autoPunctuation(w io.Writer, path string) error {
+	// [START speech_transcribe_auto_punctuation]
```

Comment:
If these are the first samples, we normally include the `func` line in the region tag. But, if this is consistent with other Go samples on the same page, this is OK.

Reply:
Included `func` line.